doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md

```diff
@@ -31,4 +31,27 @@ if($asset instanceof Asset\Document) {
 ```
 
 > This feature requires Ghostscript and at least Gotenberg or LibreOffice be installed on the server.
+> To install it on Linux:
```

Comment:
Should we mention the possibility of Docker here as well?

Reply:
Feel free